video/image_writer.c

```diff
@@ -355,11 +355,7 @@ static bool write_avif(struct image_writer_ctx *ctx, mp_image_t *image,
     av_opt_set_int(avctx, "still-image", 1, AV_OPT_SEARCH_CHILDREN);
     
     AVDictionary *avd = NULL;
-    // fuck it. We'll just inline mp_set_avdict here, otherwise we have to link
-    // a ton of objects in some random test unit.
-    char **kv = ctx->opts->avif_opts;
-    for (int n = 0; kv && kv[n * 2]; n++)
-        av_dict_set(&avd, kv[n * 2 + 0], kv[n * 2 + 1], 0);
+    mp_set_avdict(&avd, ctx->opts->avif_opts);
```

Comment:
Nitpicking but you should remove the inlining that you did in the previous commit so that way you don't have to undo in this commit here.

Reply:
I don't really know how to do that without some git hacks + as of right now, all commits build with tests, and that'd break it.

Reply:
`git rebase -i master` and edit the commits as appropriate.

Reply:
so basically squash the stub commit with the one where it was inlined, right ?

Reply:
That'll work.

Reply:
done.